db/migrations/20211110150400-create-sync-state-table.js

```diff
@@ -0,0 +1,120 @@
+"use strict";
+
+const tableName = "RepoSyncStates";
+const indexName = `${tableName}_subscriptionId_repoId_idx`
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		await queryInterface.createTable(tableName, {
+			id: {
+				type: Sequelize.INTEGER,
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true
+			},
+			subscriptionId: {
```

Comment:
Should it be a foreign key with ON DELETE CASCADE flag?

Reply:
I tried to have it as a foreign key, but was having some issues with sequelize which is apparently solved in the latest version, but didn't want to take on the extra risk right now.  This can be solved at a later stage.